bin/calculateNSCRSC.r

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env Rscript
 
+# R scripts for calculating NSC and RSC based on output files from phantompeakqualtools
+# Version 1.0
+# Author chuan-wang@github
```

Comment:
I think it's probably better to use either your e-mail address or `@chuan-wang https://github.com/chuan-wang`